safekeeper/src/timelines_global_map.rs

```diff
@@ -190,6 +198,9 @@ impl GlobalTimelines {
             Ok(_) => {
                 // We are done with bootstrap, release the lock, return the timeline.
                 drop(shared_state);
+                timeline
```

Comment:
It's probably better to send wal_backup notififcation right in the `bootstrap()`.